### PR TITLE
Adding Law of Proximity to Chance

### PR DESCRIPTION
**Pull request by josues**

### CODE REVIEW

**Comment by josues** [comment]
Also now requesting to add Law of Similarity to Chance

**Comment by josues** [comment]
And last but not least requesting to add Law of Praegnanz to Chance

File: chance/p5js/chance-praegnanz/index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---  
+layout: p5js
+author: [josues]
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance-praegnanz/index.md]: No `[]` and if you like use your real name.

File: chance/p5js/chance-praegnanz/index.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+author-url: [https://incom.org/profil/7029]
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance-praegnanz/index.md]: No `[]` please

File: chance/p5js/chance-praegnanz/index.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+title: [Law of Praegnanz and chance]
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance-praegnanz/index.md]: No `[]` please

File: chance/p5js/chance-praegnanz/index.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+tags: [P5.js,[CHANCE], [PROXIMITY], Law of Praegnanz]
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance-praegnanz/index.md]: No `[]` please  around the inner tags. Also no UPPERCASE CHANCE and PROXIMITY

File: chance/p5js/chance-proximity/index.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+---  
+layout: p5js
+author: [josues]
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance-proximity/index.md]: See the remarks on the other index.md

File: chance/p5js/chance-similartiy/index.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+---  
+layout: p5js
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance-similartiy/index.md]: see the remarks on the other index.md

**Comment by josues** [comment]
Done.

File: chance/p5js/chance-praegnanz/index.md [hunk 7]
```diff
@@ -1,11 +1,11 @@
...
-author: [josues]
-author-url: [https://incom.org/profil/7029]
-title: [Law of Praegnanz and chance]
-tags: [P5.js,[CHANCE], [PROXIMITY], Law of Praegnanz]
-permalink: /[CHANCE]/[chance-praegnanz]/
-summary: [This creates random Praegnanzen between ellipses.]
+author: josues
+author-url: https://incom.org/profil/7029
+title: Law of praegnanz and chance
+tags: P5.js, chance, proximity, Law of praegnanz
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance-praegnanz/index.md]: Sorry. Tags need the `[]` Take a look at https://raw.githubusercontent.com/fabianmoronzirfas/gestalten-in-code/master/grid/p5js/lines/index.md

File: chance/p5js/chance-proximity/index.md [hunk 8]
```diff
@@ -1,11 +1,11 @@
...
-author: [josues]
-author-url: [https://incom.org/profil/7029]
-title: [Law of proximity and chance]
-tags: [P5.js,[CHANCE], [PROXIMITY], Law of Proximity]
-permalink: /[CHANCE]/[chance-proximity]/
-summary: [This creates random proximities between ellipses.]
+author: josues
+author-url: https://incom.org/profil/7029
+title: Law of proximity and chance
+tags: P5.js, chance, proximity, Law of proximity
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance-proximity/index.md]: The same here the outer `[]` are needed

File: chance/p5js/chance-similartiy/index.md [hunk 9]
```diff
@@ -1,11 +1,11 @@
...
-author: [josues]
-author-url: [https://incom.org/profil/7029]
-title: [Law of Similarity and chance]
-tags: [P5.js,[CHANCE], [SIMILARITY], Law of Similarity]
-permalink: /[CHANCE]/[chance-similarity]/
-summary: [This creates random similarities between ellipses by varying their size and color.]
+author: josues
+author-url: https://incom.org/profil/7029
+title: Law of similarity and chance
+tags: P5.js, chance, similarity, Law of similarity
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance-similartiy/index.md]: Same here

File: chance/p5js/chance-similartiy/index.md [hunk 10]
```diff
@@ -1,11 +1,11 @@
...
+permalink: /chance/chance-similarity/
+summary: This creates random similarities between ellipses by varying their size and color.
 ---  
 
-![](https://raw.githubusercontent.com/josues/gestalten-in-code/master/chance/p5js/chance-similarity/out.png)  
+![](https://raw.githubusercontent.com/josues/gestalten-in-code/master/chance/p5js/chance-similartiy/out.png)
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance-similartiy/index.md]: Please no hardcoded urls,  If you want to refer to an image next to yours relativ markdown syntax  


    alt text  

But the code from your index.js will be integrated into the final site.

File: chance/p5js/chance-proximity/index.md [hunk 11]
```diff
@@ -1,11 +1,11 @@
...
+permalink: /chance/chance-proximity/
+summary: This creates random proximities between ellipses.
 ---  
 
 ![](https://raw.githubusercontent.com/josues/gestalten-in-code/master/chance/p5js/chance-proximity/out.png)
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance-proximity/index.md]: Please no hardcoded urls,  If you want to refer to an image next to yours relativ markdown syntax  


    alt text  

But the code from your index.js will be integrated into the final site.

File: chance/p5js/chance-praegnanz/index.md [hunk 12]
```diff
@@ -1,11 +1,11 @@
...
+permalink: /chance/chance-praegnanz/
+summary: This creates random Praegnanzen between ellipses.
 ---  
 
 ![](https://raw.githubusercontent.com/josues/gestalten-in-code/master/chance/p5js/chance-praegnanz/out.png)
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance-praegnanz/index.md]: Please no hardcoded urls,  If you want to refer to an image next to yours relativ markdown syntax  


    alt text  

But the code from your index.js will be integrated into the final site.

**Comment by josues** [comment]
What about now?

File: chance/p5js/chance-praegnanz/index.md [hunk 13]
```diff
@@ -3,9 +3,9 @@ layout: p5js
 author: josues
 author-url: https://incom.org/profil/7029
 title: Law of praegnanz and chance
-tags: P5.js, chance, proximity, Law of praegnanz
+tags: [P5.js, chance, proximity, Law of praegnanz]
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance-praegnanz/index.md]: Nearly there the laws are written like titles. For the " Law of Prägnanz" I also used the "umlaute".

File: chance/p5js/chance-praegnanz/index.md [hunk 14]
```diff
@@ -3,9 +3,9 @@ layout: p5js
...
 permalink: /chance/chance-praegnanz/
 summary: This creates random Praegnanzen between ellipses.
 ---  
 
-![](https://raw.githubusercontent.com/josues/gestalten-in-code/master/chance/p5js/chance-praegnanz/out.png)  
+![](./out.png)
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance-praegnanz/index.md]: Thanks. Having a relativ path is better. But the code from the index.js will be added automagically to the site. So there is no need to repeat the image below the actual canvas.

File: chance/p5js/chance-proximity/index.md [hunk 15]
```diff
@@ -3,9 +3,9 @@ layout: p5js
 author: josues
 author-url: https://incom.org/profil/7029
 title: Law of proximity and chance
-tags: P5.js, chance, proximity, Law of proximity
+tags: [P5.js, chance, proximity, Law of proximity]
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance-proximity/index.md]: Please write the law like this:  

    Law of Proximity

File: chance/p5js/chance-proximity/index.md [hunk 16]
```diff
@@ -3,9 +3,9 @@ layout: p5js
...
 permalink: /chance/chance-proximity/
 summary: This creates random proximities between ellipses.
 ---  
 
-![](https://raw.githubusercontent.com/josues/gestalten-in-code/master/chance/p5js/chance-proximity/out.png)  
+![](./out.png)
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance-proximity/index.md]: Same as for the first sketch. The image is not needed for the site. It is more for the download. So people don't habe to run every sketch to see the output. The code of the index.js will be added to the page.

File: chance/p5js/chance-similartiy/index.md [hunk 17]
```diff
@@ -3,9 +3,9 @@ layout: p5js
 author: josues
 author-url: https://incom.org/profil/7029
 title: Law of similarity and chance
-tags: P5.js, chance, similarity, Law of similarity
+tags: [P5.js, chance, similarity, Law of similarity]
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance-similartiy/index.md]: Law od Similarity

File: chance/p5js/chance-similartiy/index.md [hunk 18]
```diff
@@ -3,9 +3,9 @@ layout: p5js
...
 permalink: /chance/chance-similarity/
 summary: This creates random similarities between ellipses by varying their size and color.
 ---  
 
-![](https://raw.githubusercontent.com/josues/gestalten-in-code/master/chance/p5js/chance-similartiy/out.png)  
+![](./out.png)
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance-similartiy/index.md]: Same here for the image.

**Comment by josues** [comment]
Now?

File: chance/p5js/chance-praegnanz/index.js [hunk 19]
```diff
@@ -0,0 +1,40 @@
...
+var xStep;
+var yStep;
+
+var heightoff;
+var widthoff;
+
+var positions = [];
+
+function setup(){
+  createCanvas(500, 500);
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance-praegnanz/index.js]: You need to assign the canvas to a specific element so the site can render it directly.  Like this:  


```js
  var canvas = createCanvas(500, 500); // !this is important
  canvas.parent('sketch'); // !this as well
```

File: chance/p5js/chance-proximity/index.js [hunk 20]
```diff
@@ -0,0 +1,39 @@
...
+var xStep;
+var yStep;
+
+var heightoff;
+var widthoff;
+
+var positions = [];
+
+function setup(){
+  createCanvas(500, 500);
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance-proximity/index.js]: Same here:  

```js
  var canvas = createCanvas(500, 500); // !this is important
  canvas.parent('sketch'); // !this as well
```

File: chance/p5js/chance-similartiy/index.js [hunk 21]
```diff
@@ -0,0 +1,40 @@
...
+var xStep;
+var yStep;
+
+var heightoff;
+var widthoff;
+
+var positions = [];
+
+function setup(){
+  createCanvas(500, 500);
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance-similartiy/index.js]: Same for this one:  

```js
  var canvas = createCanvas(500, 500); // !this is important
  canvas.parent('sketch'); // !this as well
```

Comment (josues) [inline chance/p5js/chance-similartiy/index.js]: Done.

File: chance/p5js/chance-praegnanz/index.md [hunk 22]
```diff
@@ -0,0 +1,10 @@
+---  
+layout: p5js
+author: josues
+author-url: https://incom.org/profil/7029
+title: Law of praegnanz and chance
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance-praegnanz/index.md]: I find the title a bit generic. Can you be a bit more explicit?

File: chance/p5js/chance-praegnanz/index.md [hunk 23]
```diff
@@ -0,0 +1,10 @@
...
+tags: [P5.js, chance, proximity, Law of Prägnanz]
+permalink: /chance/chance-praegnanz/
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance-praegnanz/index.md]: same here for the foldername

File: chance/p5js/chance-praegnanz/index.md [hunk 24]
```diff
@@ -0,0 +1,10 @@
...
+summary: This creates random Praegnanzen between ellipses.
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance-praegnanz/index.md]: Also very generic

File: chance/p5js/chance-proximity/index.md [hunk 25]
```diff
@@ -0,0 +1,9 @@
+---  
+layout: p5js
+author: josues
+author-url: https://incom.org/profil/7029
+title: Law of proximity and chance
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance-proximity/index.md]: Same thing here for the title. To generic

File: chance/p5js/chance-proximity/index.md [hunk 26]
```diff
@@ -0,0 +1,9 @@
...
+tags: [P5.js, chance, proximity, Law of Proximity]
+permalink: /chance/chance-proximity/
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance-proximity/index.md]: and the folder/url also

File: chance/p5js/chance-similartiy/index.md [hunk 27]
```diff
@@ -0,0 +1,10 @@
+---  
+layout: p5js
+author: josues
+author-url: https://incom.org/profil/7029
+title: Law of similarity and chance
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance-similartiy/index.md]: Also generic

File: chance/p5js/chance-similartiy/index.md [hunk 28]
```diff
@@ -0,0 +1,10 @@
...
+tags: [P5.js, chance, similarity, Law of Similarity]
+permalink: /chance/chance-similarity/
```
Comment (fabianmoronzirfas) [inline chance/p5js/chance-similartiy/index.md]: Same here.

File: projects/analogue-paths/LICENSE [hunk 29]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 josues
```
Comment (fabianmoronzirfas) [inline projects/analogue-paths/LICENSE]: The license should have your full name.

File: projects/analogue-paths/index.md [hunk 30]
```diff
@@ -0,0 +1,70 @@
...
+authors: 
+  - Joshua Pacheco
+authors-url: 
+  - "http://incom.org/profil/7029"
+layout: project
+title: "Paths: An exploration of analogue algorithms"
+type: project
+---
+
+# PATHS
```
Comment (fabianmoronzirfas) [inline projects/analogue-paths/index.md]: Please start at H2 the H1 Headline will be added by the system https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-Projects#markdown-files--syntax

File: projects/analogue-paths/index.md [hunk 31]
```diff
@@ -0,0 +1,70 @@
...
+
+For the algorithms first iteration pencils and sheets of paper were handed to 5 people. They were instructed to consequently follow the rules of the algorithm listed below.
+
+### Iteration 1:
+Draw a rectangular form that is larger than half of the sheet. Draw another form into the first form, that is larger than half of the this first form and one that is smaller. Draw another two forms into each of the previous forms, one of them smaller and one larger than the previous form. Repeat until the sheet is full.
+
+The results turned out to be rough and did not fulfil the visual expectations.
+Therefore in the following steps the algorithm has continuously been adjusted to the impressions of natural and organic structures.
+
+![](https://raw.githubusercontent.com/josues/paths/master/assets/visuals/inspiration/Endpräsentation.004.jpeg)
```
Comment (fabianmoronzirfas) [inline projects/analogue-paths/index.md]: Don't use full URLs. These point to your repo and not the the project repo. As the Wiki states. Images should be placed in a folder called `./assets/images/` also the filenames are not url save. They should be lowercase and have no `öäü` in it. even the `.` should be removed. See https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-Projects#structure

File: projects/analogue-paths/index.md [hunk 32]
```diff
@@ -0,0 +1,70 @@
...
+![](https://raw.githubusercontent.com/josues/paths/master/assets/visuals/inspiration/Endpräsentation.007.jpeg)
```
Comment (fabianmoronzirfas) [inline projects/analogue-paths/index.md]: Same

File: projects/analogue-paths/index.md [hunk 33]
```diff
@@ -0,0 +1,70 @@
...
+![](https://raw.githubusercontent.com/josues/paths/master/assets/visuals/inspiration/Endpräsentation.009.jpeg)
```
Comment (fabianmoronzirfas) [inline projects/analogue-paths/index.md]: Same

File: projects/analogue-paths/index.md [hunk 34]
```diff
@@ -0,0 +1,70 @@
...
+![](https://raw.githubusercontent.com/josues/paths/master/assets/visuals/inspiration/Endpräsentation.012.jpeg)
```
Comment (fabianmoronzirfas) [inline projects/analogue-paths/index.md]: Same

File: projects/analogue-paths/index.md [hunk 35]
```diff
@@ -0,0 +1,70 @@
...
+
+
+### Iteration 2:
+Draw a path from one border of your sheet to another border. Draw another path starting at one point of your first path. Again conduct the path to a border of the sheet. Draw another path starting at a point of your newest Path. Conduct the path to a border of the sheet, without cutting one of your previous paths. Continue as long as possible.
+
+This text was handed to 10 people. All of them were using the same pencil and a square sheet of paper, while following the algorithms rules.
+
+![](https://raw.githubusercontent.com/josues/paths/master/assets/visuals/it2_a.jpg)
```
Comment (fabianmoronzirfas) [inline projects/analogue-paths/index.md]: Same thing for all images

File: projects/deconstructing-storr/LICENSE [hunk 36]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 josues
```
Comment (fabianmoronzirfas) [inline projects/deconstructing-storr/LICENSE]: License should have your full name

File: projects/deconstructing-storr/index.md [hunk 37]
```diff
@@ -0,0 +1,73 @@
...
+  - Joshua Pacheco
+  - Theodor Hillmann
+authors-url: 
+  - "https://incom.org/profil/7029"
+layout: project
+title: "Deconstructing Storr"
+type: project
+---
+
+Deconstructing Master Artists: Marcel Storr
```
Comment (fabianmoronzirfas) [inline projects/deconstructing-storr/index.md]: If this is the title remove it and add it to the frontmatter

File: projects/deconstructing-storr/index.md [hunk 38]
```diff
@@ -0,0 +1,73 @@
...
+
+This work is part of a class at the University of Applied Sciences Potsdam. The project aims to replicate an artists work using processing code.
+
+![](https://raw.githubusercontent.com/josues/storr/master/assets/visuals/DOKU_Marcel-Storr.002.jpeg)
```
Comment (fabianmoronzirfas) [inline projects/deconstructing-storr/index.md]: Same thing here for the images.

File: projects/deconstructing-storr/index.md [hunk 39]
```diff
@@ -0,0 +1,73 @@
...
+![](https://raw.githubusercontent.com/josues/storr/master/assets/visuals/DOKU_Marcel-Storr.007.jpeg)
+![](https://raw.githubusercontent.com/josues/storr/master/assets/visuals/DOKU_Marcel-Storr.009.jpeg)
+![](https://raw.githubusercontent.com/josues/storr/master/assets/visuals/DOKU_Marcel-Storr.010.jpeg)
+
+The second period, reaching from 1964 to 1969, is abundant in large sized drawings of palace- and cathedral-like buildings. These no longer realistic, but completely imaginary, grotesquely exaggerated into sublime and monstrous architectural visions, were sketched out with a pencil, reworked with ink and finally colorised. 
+
+End of the 1960s Storr’s wife used his absence to introduce the work to the collectors Liliane and Bertrand Kempf. She showed them his pictures, which he had hidden under the kitchen table. Since then the Kempfs traced his work even though he categorically rejected to sell his works. It lasted until 2001 until Storr’s work was shown to the public for the first time.
+
+
+Deconstructing:
```
Comment (fabianmoronzirfas) [inline projects/deconstructing-storr/index.md]: Should be H2 or H3. The whole file needs some markdown formatting

File: projects/deconstructing-storr/index.md [hunk 40]
```diff
@@ -0,0 +1,73 @@
...
+
+First approaches to deconstructing Marcel Storr’s work were taken by searching for patterns in its color palette. Therefore the dominant color and the color palettes of individual art works was determined using lokeshdhakar.com/projects/color-thief. The determined RGB color values were compared within different artworks and then assigned to there place in the color circle, using color.adobe.com, to unsuccessfully look for similar patterns within his works.
+![](https://raw.githubusercontent.com/josues/storr/master/assets/visuals/DOKU_Marcel-Storr.011.jpeg)
```
Comment (fabianmoronzirfas) [inline projects/deconstructing-storr/index.md]: images…

File: projects/deconstructing-storr/index.md [hunk 41]
```diff
@@ -0,0 +1,73 @@
...
+![](https://raw.githubusercontent.com/josues/storr/master/assets/visuals/DOKU_Marcel-Storr.014.jpeg)
+![](https://raw.githubusercontent.com/josues/storr/master/assets/visuals/DOKU_Marcel-Storr.015.jpeg)
+![](https://raw.githubusercontent.com/josues/storr/master/assets/visuals/DOKU_Marcel-Storr.016.jpeg)
+
+After the search for correlating colors did not come to an end, formal and structural patterns were observed: the buildings' strong upwards growing, their fractal unfolding, the noisy background and the detailed slenderness of the buildings’ components.
+
+![](https://raw.githubusercontent.com/josues/storr/master/assets/visuals/DOKU_Marcel-Storr.017.jpeg)
+
+
+Reconstruction:
```
Comment (fabianmoronzirfas) [inline projects/deconstructing-storr/index.md]: formatting

File: projects/deconstructing-storr/index.md [hunk 42]
```diff
@@ -0,0 +1,73 @@
...
+In a last iteration of the code, the ouptput was much more variable, creating a new ensemble of buildings every time.
+
+![](https://raw.githubusercontent.com/josues/storr/master/assets/visuals/DOKU_Marcel-Storr.025.jpeg)
+![](https://raw.githubusercontent.com/josues/storr/master/assets/visuals/DOKU_Marcel-Storr.027.jpeg)
+![](https://raw.githubusercontent.com/josues/storr/master/assets/visuals/DOKU_Marcel-Storr.028.jpeg)
+![](https://raw.githubusercontent.com/josues/storr/master/assets/visuals/DOKU_Marcel-Storr.029.jpeg)
+![](https://raw.githubusercontent.com/josues/storr/master/assets/visuals/DOKU_Marcel-Storr.030.jpeg)
+![](https://raw.githubusercontent.com/josues/storr/master/assets/visuals/DOKU_Marcel-Storr.031.jpeg)
+
+The code can be reviewed [here](https://gist.github.com/t0dorakis/0f3210240047aa7d61515d515f49bc95).
```
Comment (fabianmoronzirfas) [inline projects/deconstructing-storr/index.md]: Please create a proper repo for the code. Not just a gist.

File: projects/deconstructing-storr/index.md [hunk 43]
```diff
@@ -0,0 +1,73 @@
...
+The codes third iteration was actually building small fractal ‚houses’. The fourth adopted the idea of fractal, growing ‘houses‘ but replicated them in a assignable width and height. Putting several of the multiplied fractal ‚houses’ into one picture created the impression of one of Storr’s cathedrals.
+
+![](https://raw.githubusercontent.com/josues/storr/master/assets/visuals/DOKU_Marcel-Storr.021.jpeg)
+![](https://raw.githubusercontent.com/josues/storr/master/assets/visuals/DOKU_Marcel-Storr.022.jpeg)
+![](https://raw.githubusercontent.com/josues/storr/master/assets/visuals/DOKU_Marcel-Storr.024.jpeg)
+
+The code so far was rather static and created very similar output. 
+In a last iteration of the code, the ouptput was much more variable, creating a new ensemble of buildings every time.
+
+![](https://raw.githubusercontent.com/josues/storr/master/assets/visuals/DOKU_Marcel-Storr.025.jpeg)
```
Comment (fabianmoronzirfas) [inline projects/deconstructing-storr/index.md]: images…

File: chance/p5js/chance-praegnanz/index.md (hunk 22 above)
Comment (josues) [inline chance/p5js/chance-praegnanz/index.md]: I hope it is better now

File: chance/p5js/chance-praegnanz/index.md (hunk 23 above)
Comment (josues) [inline chance/p5js/chance-praegnanz/index.md]: I hope it is better now

File: chance/p5js/chance-praegnanz/index.md (hunk 24 above)
Comment (josues) [inline chance/p5js/chance-praegnanz/index.md]: I hope it is better now

File: chance/p5js/chance-proximity/index.md (hunk 25 above)
Comment (josues) [inline chance/p5js/chance-proximity/index.md]: I hope it is better now

File: chance/p5js/chance-proximity/index.md (hunk 26 above)
Comment (josues) [inline chance/p5js/chance-proximity/index.md]: I hope it is better now

File: chance/p5js/chance-similartiy/index.md (hunk 27 above)
Comment (josues) [inline chance/p5js/chance-similartiy/index.md]: Same here

File: chance/p5js/chance-similartiy/index.md (hunk 28 above)
Comment (josues) [inline chance/p5js/chance-similartiy/index.md]: And here

File: projects/analogue-paths/LICENSE (hunk 29 above)
Comment (josues) [inline projects/analogue-paths/LICENSE]: Done

File: projects/analogue-paths/index.md (hunk 30 above)
Comment (josues) [inline projects/analogue-paths/index.md]: Done

File: projects/analogue-paths/index.md (hunk 31 above)
Comment (josues) [inline projects/analogue-paths/index.md]: Done

File: projects/analogue-paths/index.md (hunk 32 above)
Comment (josues) [inline projects/analogue-paths/index.md]: Done

File: projects/analogue-paths/index.md (hunk 33 above)
Comment (josues) [inline projects/analogue-paths/index.md]: Done

File: projects/analogue-paths/index.md (hunk 34 above)
Comment (josues) [inline projects/analogue-paths/index.md]: Done

File: projects/analogue-paths/index.md (hunk 35 above)
Comment (josues) [inline projects/analogue-paths/index.md]: Done

File: projects/deconstructing-storr/LICENSE (hunk 36 above)
Comment (josues) [inline projects/deconstructing-storr/LICENSE]: Done

File: projects/deconstructing-storr/index.md (hunk 37 above)
Comment (josues) [inline projects/deconstructing-storr/index.md]: Done

File: projects/deconstructing-storr/index.md (hunk 38 above)
Comment (josues) [inline projects/deconstructing-storr/index.md]: Done

File: projects/deconstructing-storr/index.md (hunk 39 above)
Comment (josues) [inline projects/deconstructing-storr/index.md]: Hopefully done?

File: projects/deconstructing-storr/index.md (hunk 40 above)
Comment (josues) [inline projects/deconstructing-storr/index.md]: Done

File: projects/deconstructing-storr/index.md (hunk 41 above)
Comment (josues) [inline projects/deconstructing-storr/index.md]: Done

File: projects/deconstructing-storr/index.md (hunk 43 above)
Comment (josues) [inline projects/deconstructing-storr/index.md]: Done

File: projects/deconstructing-storr/index.md (hunk 42 above)
Comment (josues) [inline projects/deconstructing-storr/index.md]: done

**Comment by fabianmoronzirfas** [comment]
@josues and @t0dorakis Looks good to me. What about the rights for the images? Can you verify that you have the right to redistribute them? What about sources

**Pull request merged**